backend/core/confload/confload.py

```diff
@@ -16,7 +17,7 @@
 
 
 class Config:
-    def __init__(self, config_filename=None):
+    def __init__(self, config_filename=None, search_tfsm=True):
         if config_filename is None:
             config_filename = DEFAULT_FILENAME
 
```

Comment:
TL;DR: instead of depending on TFSM being located somewhere, just go find it.  That can be tricky when you don't control the environment but as long as we're explicitly targeting a containers, then that's a non-issue.  If/when we have to update it we can do so.  

If we *do* need to support more varied environments (some dude's freebsd computer or whatever) we should still be able to infer the location by importing the library and inspecting the module.  But for now I think this suffices, and lets us remove the symlinks (which were doing silly things on the container filesystem mapped to my windows filesystem when running tests, etc. etc.)

Reply:
Thanks, this make sense the initial solution was super hacky. This PR is a huge improvement